tests/load/locustfiles/locustfile.py

```diff
@@ -96,19 +98,22 @@ def on_stop(self) -> Any:
             gevent.kill(self.ws_greenlet)
 
     def on_ws_open(self, ws: WebSocket) -> None:
-        """Called when opening a WebSocket.
+        """Call when opening a WebSocket.
 
-        Args:
-            ws: WebSocket class object
+        Parameters
```

Comment:
This parameter format is different than what we're using in `db.py`, is this the preferred method?

(if so, we should probably file a follow-up ticket to standardize things to one format or another.)

Reply:
yeah, the py world has moved more towards the [NumpyScipydoc](https://numpydoc.readthedocs.io/en/latest/format.html) since it's also officially parsed by Sphinx while being a bit easier to read, while also providing inline type declarations.

I have it down that as these refactors occur in each module, the documentation part of the acceptance criteria states to normalize the docstrings to the single format.

Lmk if you have any preferences towards reStructuredText instead, we don't have to use the numpydoc format if you and others prefer re.